config/userdata.js.example

```diff
@@ -1,10 +1,14 @@
- // MUST CONFIGURE THE USER ARRAY AND GOOGLE MAPS API KEY.
- // YOU CAN GET A KEY HERE: https://developers.google.com/maps/documentation/javascript/get-api-key
- var userInfo = {
- 	users: ["username"],
- 	userZoom: false,
- 	userPath: true,
- 	zoom: 16,
- 	userFollow: true,
- 	gMapsAPIKey: "YOUR_API_KEY_HERE"
- }; 
+// MUST CONFIGURE THE USER ARRAY AND GOOGLE MAPS API KEY.
+// YOU CAN GET A KEY HERE: https://developers.google.com/maps/documentation/javascript/get-api-key
+var userInfo = {
+	users: ["username"],
+	userZoom: true,
+	zoom: 16,
+	userFollow: true,
+	gMapsAPIKey: "YOUR_API_KEY_HERE"
+//milliseconds
+var dataUpdates = {

```

Comment:
This is better to be a default value since we don't want to user report `broken`.